openeo_fastapi/client/collections.py

```diff
@@ -13,27 +11,61 @@ async def get_collections():
     """
     Basic metadata for all datasets
     """
+    stac_url = (
+        app_settings.STAC_API_URL
+        if app_settings.STAC_API_URL.endswith("/")
+        else app_settings.STAC_API_URL + "/"
+    )
+
+    try:
+        async with aiohttp.ClientSession() as client:
+            async with client.get(stac_url + "collections") as response:
+                resp = await response.json()
+                if response.status == 200 and resp.get("collections"):
+                    collections_list = []
+                    for collection_json in resp["collections"]:
+                        # For the collections from STAC, only let them through if they're on the whitelist
+                        # This has to be before the legacy collections are added.
+                        if (
+                            len(app_settings.STAC_COLLECTIONS_WHITELIST) < 1
```

Comment:
Empty lists evaulate as false, so tidier would be
```suggestion
                           not app_settings.STAC_COLLECTIONS_WHITELIST
```